jest.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+	collectCoverageFrom: [
+		'packages/**/*.{js,jsx}',
+		'!**/{node_modules,lib,coverage,scripts}/**',
+	],
+	coverageDirectory: '<rootDir>/coverage',
```

Comment:
yeah was sort of wondering about this - in theory it would be nice to get back to a level of maturity where we have a handle on test coverage, but we're pretty far away from that at the moment. Maybe best to remove